commands/operator-sdk/cmd/new.go

```diff
@@ -288,9 +292,9 @@ func doHelmScaffold() error {
 			ChartName: chart.GetMetadata().GetName(),
 		},
 		&scaffold.ServiceAccount{},
-		&scaffold.Role{IsClusterScoped: isClusterScoped},
-		&scaffold.RoleBinding{IsClusterScoped: isClusterScoped},
-		&helm.Operator{IsClusterScoped: isClusterScoped},
+		&scaffold.Role{IsClusterScoped: isClusterScopedOperator},
+		&scaffold.RoleBinding{IsClusterScoped: isClusterScopedOperator},
+		&helm.Operator{IsClusterScoped: isClusterScopedOperator},
 		&scaffold.CRD{Resource: resource},
```

Comment:
This also should receive the `isClusterScoped` variable correct?

Reply:
@estroz ah yes, indeed!

Reply:
fixed in 543b7d0